File: patterns/Python/pattern13.py
"""
Generate the following pattern
A B C D E 
A B C D
A B C
A B 
A 
"""

def pattern12(n):
    for i in range(n):
        for j in range(n-i):
            print(chr(65+j), end=" ")
            
        print("")
   
#Driver Code 
def main():
    n = int(input("Enter the number of rows: "))
    print("Here is the generated pattern\n")
    pattern12(n)
    
if __name__ == "__main__":
    main()
           

File: patterns/C++/pattern13.cpp
/*
Generate the following pattern
A B C D E 
A B C D
A B C
A B 
A  
*/

#include<bits/stdc++.h>
using namespace std;

void pattern13(int n){
    for(int i=0 ; i<n; i++){
        for(char j='A'; j<'A'+(n-i); j++){
            cout << j <<" ";
        }

        cout << endl;
    }

}


int main(){
    int n;
    cout << "Enter the Number of rows: ";
    cin >> n;

    cout << "\nHere is the generated pattern\n";
    pattern13(n);
}